src/Surfnet/StepupRa/RaBundle/Service/RecoveryTokenService.php

```diff
@@ -57,7 +58,7 @@ public function revoke(RevokeRecoveryTokenCommand $command): bool
         return $result->isSuccessful();
     }
 
-    public function search(SearchRecoveryTokensCommand $command): RecoveryTokenCollection
+    public function search(SearchRecoveryTokensCommand $command): CollectionDto
```

Comment:
This change is related to this change. Was RA the only consumer of that method?

https://github.com/OpenConext/Stepup-Middleware-clientbundle/pull/97/commits/1bad94317f397faba3c910e00acb422015fb23b8

Reply:
Not sure, but since the you calle the parent::method in the child collection, always a CollectionDto is returned